tee-worker/ts-tests/integration-tests/di_evm_identity.test.ts

```diff
@@ -340,8 +271,6 @@ describe('Test Identity (evm direct invocation)', function () {
         }
 
         assert.equal(deactivatedIdentityEvents.length, 2);
```

Comment:
Is asserting just the length sufficient here? If it's not enough, you can add a "todo" to assign it to the development team responsible for ts-tests.

Reply:
Yes for parachain events it's enough